bindings/rsac-go/bridge.c

```
@@ -0,0 +1,18 @@
+/*
+ * bridge.c — implementation of the cgo bridge shims declared in bridge.h.
+ *
+ * goAudioCallback is the Go function exported via //export in callback.go; cgo
+ * emits its prototype into _cgo_export.h, which we include here.
+ */
+
+#include "_cgo_export.h"
+#include "bridge.h"
+
+rsac_error_t rsac_go_set_callback(RsacCapture* capture, uintptr_t handle) {
```

Cppcheck (2.20.0)

[style] The function 'rsac_go_set_callback' is never used.

(unusedFunction)